pkgs/applications/misc/keepassx/community.nix

```diff
@@ -52,17 +54,21 @@ stdenv.mkDerivation rec {
 
   patches = [
     ./darwin.patch
+    ./quazip5.patch
   ];
 
   cmakeFlags = [
     "-DKEEPASSXC_BUILD_TYPE=Release"
     "-DWITH_GUI_TESTS=ON"
     "-DWITH_XC_AUTOTYPE=ON"
+    "-DWITH_XC_UPDATECHECK=OFF"
     "-DWITH_XC_YUBIKEY=ON"
-    "-DWITH_XC_KEESHARE=ON"
   ]
   ++ (optional withKeePassBrowser "-DWITH_XC_BROWSER=ON")
-  ++ (optional withKeePassHTTP "-DWITH_XC_HTTP=ON")
+  ++ (optional withKeePassKeeShare "-DWITH_XC_KEESHARE=ON")
+  ++ (optional withKeePassKeeShare "-DQUAZIP_INCLUDE_DIR=${quazip}/include/quazip")
+  ++ (optional withKeePassKeeShare "-DQUAZIP_LIBRARIES=${quazip}/lib/libquazip.so")
```

Comment:
I have not had time to test this yet. Is this compatible with OSX?

Would it be possible to 'automatically' find this using `pkg-config` by adding `quazip` to the `buildInputs`? It seems keepass does have such functionality: https://github.com/keepassxreboot/keepassxc/blob/b59fd6d06ad0b130c00c8afe8e003999335afd21/cmake/FindQuaZip.cmake

Reply:
Updated to use `pkg-config`. That required an additional line in the patch, but it's probably cleaner overall.